src/js/actions/libraries.js

```diff
@@ -620,9 +621,15 @@ define(function (require, exports) {
             .bind(this)
             .then(function (path) {
                 var layerRef = layerEffectAdapter.referenceBy.current,
-                    placeObj = layerEffectAdapter.applyLayerStyleFile(layerRef, path);
-
-                return descriptor.playObject(placeObj);
+                    placeObj = layerEffectAdapter.applyLayerStyleFile(layerRef, path),
```

Comment:
On second thought, please run these by Seth and Dave to see if we should apply library assets to locked layers, in which case these would be no-ops.